amundsen_application/static/js/components/ProfilePage/index.tsx

```diff
@@ -0,0 +1,178 @@
+      title: 'Frequently Uses (0)',
+    });
+    tabInfo.push({
+      content: this.createEmptyTabMessage('User has no bookmarked resources.'),
+      key: 'bookmarks_tab',
+      title: 'Bookmarks (0)',
+    });
+    tabInfo.push({
+      content: this.createEmptyTabMessage('User has no owned resources.'),
+      key: 'owner_tab',
+      title: 'Owner (0)',
+    });
+
+    return tabInfo;
+  }
+
+  generatePageContent = () => {
+    const user = this.state.user;
+    return (
+      <div className="container profile-page">
+        <div className="profile-header">
+            <div className="profile-avatar">
+              {
+                // default Avatar looks a bit jarring -- intentionally not rendering if no display_name
+                user.display_name && user.display_name.length > 0 &&
+                <Avatar name={user.display_name} size={74} round={true} />
+              }
+            </div>
+            <div className="profile-details">
+              <div className="profile-title">
+                <h1>{ user.display_name }</h1>
+                {
+                  (user.is_active === false) &&
+                  <Flag caseType="sentenceCase" labelStyle="label-danger" text="Alumni"/>
+                }
+              </div>
+              <text>{ `${user.role_name} on ${user.team_name}` }</text>
+              <text>{ `Manager: ${user.manager_name}` }</text>
+              <div className="profile-icons">
+                {
+                  user.is_active &&
+                  <a href={user.slack_url} className='btn btn-flat-icon-dark' target='_blank'>
+                    <img className='icon icon-slack'/>
+                    <span>Slack</span>
+                  </a>
+                }
+                {
+                  user.is_active &&
+                  <a href={`mailto:${user.email}`} className='btn btn-flat-icon-dark' target='_blank'>
+                    <img className='icon icon-mail'/>
+                    <span>{ user.email }</span>
+                  </a>
+                }
+                {
+                  user.is_active &&
+                  <a href={user.profile_url} className='btn btn-flat-icon-dark' target='_blank'>
+                    <img className='icon icon-users'/>
+                    <span>Employee Profile</span>
+                  </a>
+                }
+                <a href={`https://github.com/${user.github_name}`} className='btn btn-flat-icon-dark' target='_blank'>
+                  <img className='icon icon-github'/>
+                  <span>Github</span>
+                </a>
+              </div>
+            </div>
+        </div>
+        <div className="profile-tabs">
+          <Tabs tabs={ this.generateTabInfo() } defaultTab='frequentUses_tab' />
+        </div>
+      </div>
+    )
+  }
+
+  render() {
+    return (
+      <DocumentTitle title={ `${this.state.user.display_name} - Amundsen Profile` }>
+        <div>
+          <div className="container amundsen-crumb">
```

Comment:
The breadcrumb should be it's own component, but that can be saved for a follow up.